GhostNetwork.Profiles.Domain/SecuritySettings/SecuritySetting.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GhostNetwork.Profiles.SecuritySettings
+{
+    public class SecuritySetting
```

Comment:
In the future this model will be to big because of many AccessToSomething1, AccessToSomething2, AccessToSomething3 properties
Please group access to each section in a some new class